### PR TITLE
(refactor): Modify the Service Account names of the experiments

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>

- Modify the Service Account names of the experiments

### CODE REVIEW

File: charts/coredns/coredns-pod-delete/rbac.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: coredns-sa
+  name: coredns-pod-delete-sa
```
Comment (ispeakc0de) [inline charts/coredns/coredns-pod-delete/rbac.yaml:4]: Pls change the name of service Account in the whole file [ it have multiple occurrence here] and also change the name of Role/clusterRole & RoleBinding/CluterRoleBinding

File: charts/generic/container-kill/rbac.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: nginx-sa
+  name: container-kill-sa
```
Comment (ispeakc0de) [inline charts/generic/container-kill/rbac.yaml:5]: Also change the SA name in engine manifest

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: charts/kafka/kafka-broker-disk-failure/rbac.yaml [hunk 3]
```diff
@@ -21,15 +21,15 @@ rules:
...
 metadata:
-  name: kafka-sa
+  name: kafka-broker-disk-failure-sa
   labels:
-    name: kafka-sa
+    name: kafka-broker-disk-failure-sa
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: kafka-role
```
Comment (ispeakc0de) [inline charts/kafka/kafka-broker-disk-failure/rbac.yaml]: kafka-broker-disk-failure-sa

File: charts/kafka/kafka-broker-pod-failure/rbac.yaml [hunk 4]
```diff
@@ -23,14 +23,14 @@ rules:
...
 metadata:
-  name: kafka-sa
+  name: kafka-broker-pod-failure-sa
   labels:
-    name: kafka-sa
+    name: kafka-broker-pod-failure-sa
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: kafka-role
```
Comment (ispeakc0de) [inline charts/kafka/kafka-broker-pod-failure/rbac.yaml]: kafka-broker-pod-failure-sa

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Review by ksatchit — DISMISSED** [review]

**Review by ispeakc0de — APPROVED** [review]
/lgtm

**Pull request merged**